tests/test_core/test_learning_rate_decay_optimizer_constructor.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+
+from mmseg.core.utils.layer_decay_optimizer_constructor import \
+    LearningRateDecayOptimizerConstructor
+
+base_lr = 1000000000
+decay_rate = 9
```

Comment:
I think 2 might be better for `decay_rate`.

My suggestion is that base_lr might be `1`, and `lr_scale` will equal to `lr` and check it after line 165.